### DISCUSSION TITLE
Issue with the X icon

### DISCUSSION THREAD

**Discussion by aksophos** [primefaces/community#2444] [PrimeIcons]
Hi, I was using the pi-times icon as the X button. But just noticed that it started showing up as a pause icon instead. 

`<i class="pi pi-times"></i> `

How can I fix this?

**Comment by ibrahimkelly** [comment]
> Hi, I was using the pi-times icon as the X button. But just noticed that it started showing up as a pause icon instead.
> 
> `<i class="pi pi-times"></i> `
> 
> How can I fix this?

Can you provide a screenshot?
